### DISCUSSION TITLE
Docs aren't clear on Tex Live

### DISCUSSION THREAD

**Discussion by lachlansimpson** [quarto-dev/quarto-web#1285] [General]
The documentation for PDF engines notes:

_this engine will detect and attempt to install missing packages, fonts, or commands if TeX Live is available_

https://github.com/quarto-dev/quarto-web/blob/a96c1b924cf869912b25c5ac30b2a8558327f169/docs/output-formats/pdf-engine.qmd#L115-L117

But it isn't clear what is meant by "Tex Live". Is it sufficient to have the single package `texlive-latex` installed in RHEL/`texlive` in Ubunutu; or do all or a subset of the numerous `texlive-*` packages also need to be included? Is an installation of the package from source acceptable? https://www.tug.org/texlive/quickinstall.html

**Comment by cderv** [comment]
It really means it should work with TeX Live, no matter how it is installed. We recommend a local install for Quarto to use called `TinyTeX` that you can get with `quarto install tinytex` because we test against it, but any TeX Live installation found in PATH (searching for `tlmgr`) should work. 

The top of the page explains clearly that Quarto works with various engine, but one is preferred for best experience
https://quarto.org/docs/output-formats/pdf-engine.html#overview

On the contrary, using MikTeX for example won't allow automatic installation
